src/js.rs

```diff
@@ -80,6 +80,19 @@ extern "C" {
     /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval
     #[wasm_bindgen(catch)]
     pub fn eval(js_source_text: &str) -> Result<JsValue, JsValue>;
+
+    /// The `parseInt()` function parses a string argument and returns an integer
+    /// of the specified radix (the base in mathematical numeral systems).
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
+    #[wasm_bindgen(js_name = parseInt)]
+    pub fn parse_int(text: &str, radix: u8) -> JsValue;
+
+    /// The parseFloat() function parses an argument and returns a floating point number.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat
+    #[wasm_bindgen(js_name = parseFloat)]
+    pub fn parse_float(text: &str) -> JsValue;
```

Comment:
Should this perhaps return `f64` as I think it always returns a float?

Reply:
I tried, but ...

```
---- js_globals::parse_int_float stdout ----
···················································
running "cargo" "build" "--target" "wasm32-unknown-unknown"
exit: exit code: 0
stderr ---
   Compiling test0 v0.0.1 (file:///D:/github/other/wasm-bindgen/target/generated-tests/test0)
    Finished dev [unoptimized + debuginfo] target(s) in 0.61s

dur: 0.812s
···················································
running wasm-bindgen
dur: 1.739s
···················································
running "node" "--experimental-modules" "run.mjs"
exit: exit code: 1
stderr ---
(node:23524) ExperimentalWarning: The ESM module loader is experimental.
SyntaxError: Unexpected token import
    at ModuleJob.loaders.set [as moduleProvider] (internal/loader/ModuleRequest.js:32:13)
    at <anonymous>

thread 'js_globals::parse_int_float' panicked at 'assertion failed: output.status.success()', crates\test-project-builder\src\lib.rs:767:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
dur: 0.328s


failures:
    js_globals::parse_int_float
```

the `#[wasm_bindgen]`-ed export function is:

```
export function __wbg_f_parseFloat_parse_float_n(arg0, arg1) {
    let varg0 = getStringFromWasm(arg0, arg1);
    return parseFloat(varg0);  // here returns a js `number`, not `f64`
}
```